operators/multiclusterobservability/controllers/multiclusterobservability/observatorium.go

```
@@ -982,6 +982,34 @@ func newCompactSpec(mco *mcov1beta2.MultiClusterObservability, scSelected string
+	}
+	if mco.Spec.AdvancedConfig != nil && mco.Spec.AdvancedConfig.Compact != nil && mco.Spec.AdvancedConfig.Compact.Debug != nil && mco.Spec.AdvancedConfig.Compact.Debug.WaitInterval != "" {
+		duration, err := time.ParseDuration(mco.Spec.AdvancedConfig.Compact.Debug.WaitInterval)
+		if err != nil {
+			log.Error(err, "Failed to parse wait interval", "waitInterval", mco.Spec.AdvancedConfig.Compact.Debug.WaitInterval)
+			return compactSpec
+		}
+		if duration > 5*time.Minute {
+			compactSpec.Args = append(compactSpec.Args, "--web.disable")
+		}
+	}
+	if mco.Spec.AdvancedConfig != nil && mco.Spec.AdvancedConfig.Compact != nil && mco.Spec.AdvancedConfig.Compact.Debug != nil && mco.Spec.AdvancedConfig.Compact.Debug.BlockMetaFetchConcurrency != nil {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(golines)